src/main/java/com/basho/riak/client/api/commands/kv/MultiCommand.java

```diff
@@ -371,9 +371,9 @@ public void await() throws InterruptedException
         }
 
         @Override
-        public void await(long timeout, TimeUnit unit) throws InterruptedException
+        public boolean await(long timeout, TimeUnit unit) throws InterruptedException
         {
```

Comment:
According to http://stackoverflow.com/questions/3589946/retrofitting-void-methods-to-return-its-argument-to-facilitate-fluency-breaking/3589948#3589948, this is a source-compatible, but binary-incompatible change.